tests/test_hydra.py

```diff
@@ -1067,7 +1067,12 @@ def test_app_with_error_exception_sanitized(tmpdir: Any, monkeypatch: Any) -> No
 Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace."""
 
     ret = run_with_error(cmd)
-    assert normalize_newlines(expected) == normalize_newlines(ret)
+    assert_regex_match(
+        from_line=expected,
+        to_line=ret,
+        from_name="Expected output",
+        to_name="Actual output",
+    )
```

Comment:
can you show sample output for when it fails?

Reply:
```
------------------------------------- Captured stdout call -------------------------------------

------------ DIFF -------------
--- Expected output

+++ Actual output

@@ -1,6 +1 @@

-Traceback (most recent call last):
-  File ".*my_app.py", line 9, in my_app
-    1 / 0
-ZeroDivisionError: division by zero
-
-Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
+dummy ouptput
-------------------------------
```

after https://github.com/facebookresearch/hydra/pull/1197/commits/403a99ffcb7c410ac7459c11c15fb73954433940